config/delta-producer/organizations-public-info/export.json

```diff
@@ -103,7 +102,9 @@
         "http://data.lblod.info/vocabularies/erediensten/typeEredienst",
         "http://www.w3.org/ns/adms#identifier",
         "http://www.w3.org/ns/regorg#legalName",
-        "http://www.w3.org/2004/02/skos/core#altLabel"
+        "http://www.w3.org/2004/02/skos/core#altLabel",
+        "http://www.w3.org/ns/org#hasPrimarySite",
+        "http://www.w3.org/ns/org#hasSite"
```

Comment:
**Comment (non-blocking):** Fyi, OP currently does not use the `org:hasSite` predicate for representative bodies, so adding it will not provide additional data. This may change in the future.